src/frame/content.rs

```diff
@@ -193,6 +250,43 @@ pub struct SynchronisedLyrics {
     pub content: Vec<(u32, String)>,
 }
 
+const MILLISECONDS_PER_HOUR: u32 = 3600000;
+const MILLISECONDS_PER_MINUTE: u32 = 60000;
+const MILLISECONDS_PER_SECOND: u32 = 1000;
+
+impl SynchronisedLyrics {
+    pub fn format_table(&self) -> String {
```

Comment:
`warn(missing_docs)` is triggering on this function. Could you please add some documentation?

Furthermore, could you please update this function so it writes its output to an `impl io::Write`? I think this makes it more versatile :)

Reply:
Fixed in 489e198